packages/grid/test/drag-and-drop.common.js

```diff
@@ -254,6 +254,33 @@ describe('drag and drop', () => {
         });
       });
 
+      it('should add part to the sources of dragged rows', async () => {
```

Comment:
If it makes sense, this test could be split into multiple smaller tests..for example
"should add dragged cell part names to the dragged row"
"should not add dragged cell part names to non-dragged row"
"should add dragged cell part names to the dragged rows"

Reply:
Sure!